config/services.yaml

```diff
@@ -0,0 +1,31 @@
+services:
+  _defaults:
+    autowire: true
+    autoconfigure: true
+    public: false
+
+  Skrz\Bundle\BunnyBundle\:
+    resource: "../src/*"
+    exclude: "../src/{DependencyInjection,BunnyBundle.php}"
+
+  Skrz\Bundle\BunnyBundle\Service\BunnyManager:
+    arguments:
+      $config: "%skrz_bunny%"
+
+  Bunny\Client:
+    arguments:
+      $options: "%skrz_bunny%"
+    public: true # this is because of autowirebundle
```

Comment:
Tohle je pořád kvůli autowire bundlu?

Reply:
ano je chci to odstranit az budeme mit premigrovano ale ted je to potreba